ui/src/app/utils/cdr-versions.spec.tsx

```diff
@@ -1,47 +1,90 @@
 import * as React from "react";
 
-import {Workspace, CdrVersion, CdrVersionListResponse} from 'generated/fetch';
+import {Workspace} from 'generated/fetch';
 import {WorkspaceStubVariables} from 'testing/stubs/workspaces';
-import {cdrVersionListResponse} from 'testing/stubs/cdr-versions-api-stub';
-import {getCdrVersion, getDefaultCdrVersion, hasDefaultCdrVersion} from "./cdr-versions";
+import {cdrVersionTiersResponse} from 'testing/stubs/cdr-versions-api-stub';
+import {getCdrVersion, hasDefaultCdrVersion, getDefaultCdrVersionForTier} from "./cdr-versions";
+import {AccessTierShortNames} from "./access-tiers";
 
+const registeredCdrVersionTier = cdrVersionTiersResponse.tiers[0];
```

Comment:
I suggest moving these definitions into `cdr-versions-api-stub` instead so the data definitions live in the same place, or else switching this to do a lookup by tier name - whatever lets you avoid a non-obvious dependency on this array order